Tests/test_cards.gd

```diff
@@ -89,11 +97,12 @@ func test_draw_to_max():
 
 
 func test_discard_specific():
-	_card_container.draw_cards(1)
-	var card: CardWorld = _card_container.cards_in_hand[0]
+	_card_container.draw_cards(5)
+	var card: CardWorld = _card_container.cards_in_hand[2]
+	assert_eq(card.card_data.card_title, "Card3")
```

Comment:
should also assert that `assert_eq(_card_container.cards_in_hand[0].card_title = "Card1"` (or Card0 if we go with the comment I added above). And the same for every card left in the hand

Reply:
Yeah that can be added, wouldn’t hurt. I won’t be able to push for a little bit so feel free to make the change if you’d like.

Reply:
I'll need to check if I can push on your branch

Reply:
I'm back now, just pushed this